students/kevin/session02/series.py

```diff
@@ -0,0 +1,45 @@
+def fibonacci(n):
+    """
+    Produces
+    :param n:
+    :return: 
+    """
+
+    fibList = [0, 1]
+
+    while len(fibList) < n:
+        fibList.append(fibList[-2] + fibList[-1])
+
+    return fibList[n-1]
+
```

Comment:
With fibonacci numbers, if the number is less than 0, it won't work. If a number is less than 2, the fibonacci sequence is 1. For all other numbers, this formula fibonacci(n - 1) + fibonacci(n - 2) will return the fibonacci sequence. Please think about the language of how I phrased that to help you with your flow control.